deepsensor/model/model.py

```diff
@@ -648,6 +648,597 @@ def unnormalise_pred_array(arr, **kwargs):
+                                    x1=False,
+                                )
+                                b_x2_min = (
+                                    patch_x2_index[0] - prev_patch_x2_min
+                                ) - patch_overlap[1]
+                        else:
+                            b_x2_max = b_x2_max + 1
+
+                        if patch_x1_index[0] == data_x1_index[0]:
+                            b_x1_min = 0
+                        # TODO: ensure this elif statement is robust to multiple patch sizes.
+                        elif abs(patch_x1_index[1] - data_x1_index[1]) < 2:
+                            b_x1_max = 0
+                            b_x1_max = b_x1_max + 1
+                            patch_prev = preds[i - patches_per_row]
+                            if x1_ascend:
+                                prev_patch_x1_max = get_index(
+                                    patch_prev[var_name].coords[orig_x1_name].max(),
+                                    x1=True,
+                                )
+                                b_x1_min = (
+                                    prev_patch_x1_max - patch_x1_index[0]
+                                ) - patch_overlap[0]
+                            else:
+                                prev_patch_x1_min = get_index(
+                                    patch_prev[var_name].coords[orig_x1_name].min(),
+                                    x1=True,
+                                )
+
+                                b_x1_min = (
+                                    prev_patch_x1_min - patch_x1_index[0]
+                                ) - patch_overlap[0]
+                        else:
+                            b_x1_max = b_x1_max + 1
+
+                        patch_clip_x1_min = int(b_x1_min)
+                        patch_clip_x1_max = int(
+                            data_array.sizes[orig_x1_name] - b_x1_max
+                        )
+                        patch_clip_x2_min = int(b_x2_min)
+                        patch_clip_x2_max = int(
+                            data_array.sizes[orig_x2_name] - b_x2_max
+                        )
+
+                        patch_clip = data_array.isel(
+                            **{
+                                orig_x1_name: slice(
+                                    patch_clip_x1_min, patch_clip_x1_max
+                                ),
+                                orig_x2_name: slice(
+                                    patch_clip_x2_min, patch_clip_x2_max
+                                ),
+                            }
+                        )
+
+                        patches_clipped[var_name].append(patch_clip)
+
+            combined = {
+                var_name: xr.combine_by_coords(patches, compat="no_conflicts")
+                for var_name, patches in patches_clipped.items()
+            }
```

Comment:
Can you expose an argument for the method used to combine patches (currently only `"remove_overlap"` supported, or whatever you think is more appropriate. This will make it more clear how to add new combining methods (like weighted averaging) in future.

Reply:
This method wasn't really working, it required there to be no overlap between patches, so we've taken a different approach, which doesn't allow for different joining methods at present.